receiver/splunkenterprisereceiver/scraper.go

```diff
@@ -126,7 +128,13 @@ func (s *splunkScraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 	wg.Wait()
 	close(errChan)
 	errs = <-errOut
-	return s.mb.Emit(), errs.Combine()
+	combinedErrs := errs.Combine()
+	if combinedErrs != nil {
+		s.settings.Logger.Error(fmt.Sprintf("Scrape errors: %v", combinedErrs.Error()))
+		s.failedScrapes += 1
+		s.mb.RecordSplunkenterpriseErrorDataPoint(now, s.failedScrapes)
+	}
+	return s.mb.Emit(), nil
```

Comment:
Instead of returning `combinedErrs` increment the failedScrapes count and record it asa metric.